src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,6 +130,83 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     setAnchorEl(null);
   };
 
+  const userToAutocompleteOption = (user: User): { label: string; id: string } => {
```

Comment:
id doesnt have to be a string i dont think but i cant one hundred percent remember, also we already have a function that does exactly this, we should refrain from remaking the same function in a ton of files, instead search for this function in VSCode and then either move it to a util file or use one thats already being exported